tests/test_brownian_interval.py

```diff
@@ -0,0 +1,311 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Test the `BrownianInterval``.
+
+The suite tests both running on CPU and CUDA (if available).
+"""
+import sys
+
+sys.path = sys.path[1:]  # A hack so that we always import the installed library.
+
+import numpy as np
+import numpy.random as npr
+import torch
+from scipy.stats import norm, kstest
+
+import pytest
+import torchsde
+
+torch.manual_seed(2147483647)
```

Comment:
Is there a particular reason to choose 2**31 - 1 as opposed to 0?

Reply:
Whoops, didn't mean to choose a magic number like that. (I copied and pasted it from the PyTorch docs without thinking.)

Whilst obviously this particular choice doesn't help, it was meant in response to this line from the PyTorch docs re: random seeds:
"""
It is recommended to set a large seed, i.e. a number that has a good balance of 0 and 1 bits. Avoid having many 0 bits in the seed.
"""
I don't know how much that would actually affect us, but I thought it best to follow good practice.

Reply:
SGTM.